js/foundation.core.plugin.js

```diff
@@ -44,11 +44,7 @@ function hyphenate(str) {
 }
 
 function getPluginName(obj) {
-  if(typeof(obj.constructor.name) !== 'undefined') {
```

Comment:
Not sure but I think there was some reason for this in the past (IE10 and 11?). We'll see in the tests what will happen.